static/js/emoji_picker.js

```diff
@@ -668,15 +668,10 @@ exports.register_click_handlers = function () {
         e.stopPropagation();
 
         const elem = $(e.currentTarget);
-        const title = i18n.t("Add emoji reaction");
-        elem.tooltip({
-            title: title + " (:)",
-            trigger: 'hover',
-            placement: 'bottom',
-            animation: false,
-        });
-        elem.tooltip('show');
-        $(".tooltip-arrow").remove();
+        const text = i18n.t("Add emoji reaction");
+        const title = text + " (:)";
```

Comment:
Nitpick: Can we concatenate on a single line to avoid this single use variable declaration?

Reply:
```

(zulip-py3-venv) vagrant@ubuntu-bionic:/srv/zulip$ ./tools/lint --g=frontend                                                                                                 js        \| Do not concatenate i18n strings at static/js/emoji_picker.js line 671:                                                                                           js        \|         const title = i18n.t("Add emoji reaction") + " (:)";
--


```

I had to make the change because of this. The original HTML titles didn't include the Keyboard shortcuts in the translation, so I had to keep it outside.

I've had to do similar changes in the other commits as well.

Reply:
Ahh thanks for the info, I thought that might have been the case.

Reply:
(Thought it seems a bit funny to lint against in-line concatenation, but to allow concatenation with `i18n` string variables?)